### ISSUE TITLE
Simple question on the result

### ISSUE THREAD

**Issue by mazzzystar** [limingcv/Photorealistic-Style-Transfer#1]
I think if you try to maximize the similarity between the `Generated` and the `Content` image, then the every structural texture are almost the same. Then it's only the color difference between them.

So what the different between your method with a filter that extracting target dominant hue ？

**Comment by limingcv** [comment]
For photorealistiic style transfer, we want the output image preserves the original structure and detail outline of the content image after transfer.

The most significant difference comes from visual effects，our method not only makes the output more uniform in color distribution, but also contains more semantic information of content image.

**Comment by mazzzystar** [comment]
Your generated result has almost the same semantic information with content image, so only the color is changed. I think it's reasonable if your method is trying to assign different color shade for different pixel boundary.

Below is my result of using only PhotoShop: `Image->Adjustment->Hue`


<br/><div align="center">
<img src="https://raw.githubusercontent.com/limingcv/Photorealistic-Style-Transfer/master/supplementary_material/more_results/Comic/content.png" width="31%" height="30%" align="center">
<img src="https://user-images.githubusercontent.com/6824141/56956833-dcba7c00-6b77-11e9-873b-063de5b2de88.png" width="31%" height="30%" align="center">
<img src="https://user-images.githubusercontent.com/6824141/56957593-4176d600-6b7a-11e9-9533-7f8f564db9ce.png" width="31%" height="30%" align="center">
</div>

**Comment by limingcv** [comment]
Yes, I think you are right. In this example, our model attempts to assign different hue to different pixel boundaries for good results.

**Comment by KushajveerSingh** [comment]
What about cases when our content image is in no light and we provide style image in lighted conditions. In that case, lights are actually transferred from style to content image. And I think it is difficult to reproduce that on Photoshop.
